File: Lv.1/2016년.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b) {
    
    int months[12] = {31,29,31,30,31,30,31,31,30,31,30,31};
    string result = "";
    
    int totalDay = 0;
    for(int i=0;i<a-1;i++){
        totalDay+=months[i];
    }
    totalDay+=b-1;
    
    if(totalDay%7==0){
        result="FRI";
    }else if(totalDay%7==1){
        result="SAT";
    }else if(totalDay%7==2){
        result="SUN";
    }else if(totalDay%7==3){
        result="MON";
    }else if(totalDay%7==4){
        result="TUE";
    }else if(totalDay%7==5){
        result="WED";
    }else if(totalDay%7==6){
        result="THU";
    }
    
    
    return result;
}

File: Java/LV1/2016년.java
import java.util.*;

class Solution {
    public String solution(int a, int b) {
        String answer = "";
        int[] cal = {31,29,31,30,31,30,31,31,30,31,30,31};
        String[] day = {"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};
        int total = 0;
        for(int i=0;i<a-1;i++){
            total+=cal[i];
        }
        total+=b;
        total-=1;
        // 1/1 금
        // 1/2 토
        // 1/3 일
        // 1/4 월
        // 1/5 화
        // 1/6 수
        // 1/7 목
        // 1/8 금
        
        System.out.println(total);
        int k = total%=7;
        answer = day[k];
        return answer;
    }
}

